app/views/design-system/rsd-design-system/design-patterns/contact-details/example-contact-per-card-edit.html

```diff
@@ -0,0 +1,26 @@
+{% extends "./../example-layout.html" %}
+
+{% block pageTitle %}
+Edit contact details design pattern (contact per card) | RSD component library
+{% endblock %}
+
+{% block sample %}
+<h1 class="govuk-heading-xl">
+  <span class="govuk-caption-l">Secondary title here</span>
+  Edit Project managed by
+</h1>
+<div class="govuk-form-group">
+  <div class="govuk-form-group">
+    <label class="govuk-label govuk-heading-m" for="teamLeadName">
+      Name
+    </label>
+    <input class="govuk-input" id="teamLeadName" name="teamLeadName" type="text" value="Alex James">
+  </div>
+  <div class="govuk-form-group">
+    <label class="govuk-label govuk-heading-m" for="teamLeadEmailAddress">
+      Email address
+    </label>
+    <input class="govuk-input" id="teamLeadEmailAddress" name="teamLeadEmailAddress" type="text" value="alex.james@education.gov.uk">
```

Comment:
Don't put potentially real user email addresses  into this code. remove this.

Reply:
Thanks Andy, we have replaced all @education.gov.uk emails with @example.com